solutions/instances/variables.tf

```diff
@@ -39,7 +39,7 @@ variable "existing_kms_instance_crn" {
 variable "existing_scc_cos_kms_key_crn" {
   type        = string
   default     = null
-  description = "The CRN of an existing KMS key to be used to encrypt the SCC COS bucket. If no value is passed, a value must be passed for either 'var.existing_kms_instance_crn' if you want to create a new key ring and key, or 'var.existing_scc_cos_bucket_name' if you want to use an existing bucket."
+  description = "(OPTIONAL) The CRN of an existing KMS key to be used to encrypt the SCC COS bucket. If no value is passed, a value must be passed for either the existing_kms_instance_crn input variable if you want to create a new key ring and key, or the existing_scc_cos_bucket_name input variable if you want to use an existing bucket."
```

Comment:
can you put backticks around `existing_kms_instance_crn` and `existing_scc_cos_bucket_name`?